From: Juergen Gross <jgross@suse.com>
Subject: kABI fix after KVM: x86: Convert vcpu_run()'s immediate exit
 param into a generic bitmap
Patch-mainline: Never, kABI workaround
References: git-fixes

Hide the change of exit_fastpath_completion() interface from GENKSYMS.

Signed-off-by: Juergen Gross <jgross@suse.com>
---
diff --git a/arch/x86/include/asm/kvm_host.h b/arch/x86/include/asm/kvm_host.h
--- a/arch/x86/include/asm/kvm_host.h
+++ b/arch/x86/include/asm/kvm_host.h
@@ -1753,7 +1753,11 @ struct kvm_x86_ops {
 
 	int (*vcpu_pre_run)(struct kvm_vcpu *vcpu);
 	enum exit_fastpath_completion (*vcpu_run)(struct kvm_vcpu *vcpu,
+#ifdef __GENKSYMS__
+						  bool force_immediate_exit);
+#else
 						  u64 run_flags);
+#endif
 	int (*handle_exit)(struct kvm_vcpu *vcpu,
 		enum exit_fastpath_completion exit_fastpath);
 	int (*skip_emulated_instruction)(struct kvm_vcpu *vcpu);
